CHANGELOG.md

```diff
@@ -2,6 +2,36 @@
 
 Notable changes to Apisonator will be tracked in this document.
 
+## 2.86.0 - 2018-05-10
+
+### Added
+
+- The metrics used for reporting transactions and authorizations to the master
+  account are now configurable under `master.metrics.transactions` and
+  `master.metrics.transactions_authorize`. ([#15](https://github.com/3scale/apisonator/pull/15))
+- There is now a set of rake targets for selectively run tests and specs by type
+  (ie. integration, unit, etc) and by specific file. ([#11](https://github.com/3scale/apisonator/issues/11))
+- The `make dev` command supports specifying an env variable `PORT` to expose
+  the listener in that port on the local machine. ([#8](https://github.com/3scale/apisonator/pull/8))
+- Minor miscellaneous improvements. ([#7](https://github.com/3scale/apisonator/pull/7), [#9](https://github.com/3scale/apisonator/pull/9), [#12](https://github.com/3scale/apisonator/issues/12), [#16](https://github.com/3scale/apisonator/pull/16), [#17](https://github.com/3scale/apisonator/pull/17), pr [#20](https://github.com/3scale/apisonator/pull/20), [#23](https://github.com/3scale/apisonator/pull/23), [#24](https://github.com/3scale/apisonator/pull/24))
```

Comment:
There's a `pr` there that should be deleted.

Reply:
good catch, that is a leftover